tests/e2e/high_availability_experimental/single_coordinator.py

```diff
@@ -147,6 +147,105 @@ def retrieve_data_show_replicas():
     interactive_mg_runner.stop_all(MEMGRAPH_INSTANCES_DESCRIPTION)
 
 
+def test_replication_works_on_replica_instance_restart():
```

Comment:
I think we already have exactly same test

Reply:
resolved on call